data/campaigns/Liberty/units/Maddock.cfg

```diff
@@ -0,0 +1,72 @@
+#textdomain wesnoth-l
+[unit_type]
+    id=Lord of Elensefar
+    name= _ "Lord of Elensefar"
+    race=human
+    profile="portraits/maddock.webp"
+    description=_"The grand city-state of Elensefar is the wealthiest human settlement on the Great Continent, commanding a reputation and power that belies its relatively small population. Unlike the more war-minded inland nobles, Elensefar’s governors master the art of commerce and the skill of sailing, and are known to be particularly agreeable. Yet behind that merchant charm lies a fierce independence, a trait that sometimes puts them at odds with the aristocrats of other regions."
```

Comment:
I suppose it's not a big deal if it's left as-is, but I wonder if 'belligerent' might be a better replacement for 'war-minded'.